src/main/java/com/oltpbenchmark/benchmarks/templated/procedures/GenericQuery.java

```diff
@@ -36,9 +36,16 @@ public abstract class GenericQuery extends Procedure {
 
     /** Execution method with parameters. */
     public void run(Connection conn, List<Object> params) throws SQLException {
-        try (PreparedStatement stmt = getStatement(conn, params); ResultSet rs = stmt.executeQuery()) {
-            while (rs.next()) {
-                //do nothing
+        PreparedStatement stmt = getStatement(conn, params);
```

Comment:
I would leave the creation of the stmt object in a try clause, this avoids the object being used for any other purpose (down the line).